### ISSUE TITLE
Add external supporters page

### ISSUE THREAD

**Issue by kenodressel** [option26/quarantine-hero#217]
the page should feature:
- organizations that support quarantaenehelden with resources
- friendly organizations that offer good services / resources related to corona
- individuals who supported us

**Comment by tgraupne** [comment]
We are currently still in discussion with the GroßTeam about the design, content and specific use-case of this page.

**Comment by Maurice22** [comment]
Is currently conceptualized. Once this is done we will implement it.

**Comment by mauriceackel** [comment]
Duplicate of #315 
Closing now.
